config/routes.rb

```diff
@@ -1,6 +1,11 @@
 Rails.application.routes.draw do
+  get 'messages/random_greeting'
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
 
   # Defines the root path route ("/")
   # root "articles#index"
```

Comment:
- [ ] It would be better if we define our root routes in our application as seen in the screenshot below
![Screenshot from 2023-02-21 10-04-43](https://user-images.githubusercontent.com/46388113/220312406-b7407ff4-20b4-4ba6-bd80-249278bf36fd.png)